js/views/homeVw.js

```diff
@@ -449,44 +484,32 @@ module.exports = Backbone.View.extend({
     }
   },
 
-  removeSubView: function(view) {
-    var self = this,
-        index;
+  _removeSubView: function(list, view) {
```

Comment:
Why prefix this function name with an underscore? Just curious, since it's different from our usual naming convention (not that we have a very strict one).